app/classifier/tasks/dropdown/dropdown-dialog.cjsx

```diff
@@ -141,6 +152,16 @@ DropdownDialog = React.createClass
 
     return select
 
+  handleAdd: ->
+    optionsArray = @refs.optionInput.value.split(/\n|,/)
```

Comment:
I recommend having this split strictly on lines. The results of

```
One thing
Another thing, with a comma
```

would be unexpected.